test/parallel/test-url-override-hostname.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
```

Comment:
Other test files for this API usually have a name prefix like `test-whatwg-url-`